specification/personal-demographics.yaml

```diff
@@ -1082,7 +1082,44 @@ paths:
         *	Line 4 - post town, e.g. `Leeds`
         *	Line 5 - county (if present), e.g. `West Yorkshire`.
 
-        When updating an address, lines 1 or 2, and line 4 must be populated; line 5 should not be included in vernacular addresses but may be included in PAF-derived addresses.
+        When updating an address, lines 1 or 2, and line 4 must be populated; line 5 should not be included in manually created addresses but may be included in PAF derived addresses.
+
+        Address lines should be populated otherwise the message may be rejected. There are exceptions:
+        * if a postcode and PAF key are submitted, in which case the lines will be automatically populated, however if there are no matches or too many matches they message will be rejected due to missing address lines.
+        * a pseudo postcode is used. For example `ZZ99 3VZ` meaning `no fixed abode`.
```

Comment:
can a link be put in here to the pseudo postcode list

Reply:
Typo on "they"

Reply:
Fixed typo - linking within an apigee doc apparently does not work, so cannot add a link to the same page.